setup.py

```diff
@@ -5,13 +5,14 @@
 setuptools.setup(
     name='cpg-utils',
     # This tag is automatically updated by bumpversion
-    version='4.5.1',
+    version='4.5.2',
     description='Library of convenience functions specific to the CPG',
     long_description=open('README.md').read(),
     long_description_content_type='text/markdown',
     url=f'https://github.com/populationgenomics/cpg-utils',
     license='MIT',
-    packages=['cpg_utils'],
+    packages=['cpg_utils']
+    + ['cpg_utils.' + p for p in sorted(setuptools.find_packages('./cpg_utils'))],
```

Comment:
Did @MattWellie's https://github.com/populationgenomics/cpg-utils/pull/66 not work?

Reply:
Oh sorry, my bad, I totally missed that PR. I just realised the GitHub slack bot stopped working a few days ago 😨